### ISSUE TITLE
Sepataring definition and examples to it

### ISSUE THREAD

**Issue by qstixxx** [ninja33/ODH#354]
Hello,
I'm using the Cambridge EN > CN dictionary and wonder if it is possible to separate definitions and examples into different fields when creating an Anki card. Not really into programming, so can't rewrite the code by myself.
And the other thing, is it also possible to remove all the Chinese from this dictionary and leave it in plain English when creating a card?

Much appreciate

**Comment by Thuylt185411** [comment]
https://static.staticsave.com/tracy1/241126.json
you can use my script to use En_en_cambridge
i'm finding the solution to separate each field in definitions, too
